web/api/netdata-swagger.json

```diff
@@ -1366,6 +1367,30 @@
           }
         }
       },
+      "alarms_values": {
+        "type": "object",
+        "properties": {
+          "hostname": {
+            "type": "string",
+            "description": "The hostname of the netdata server."
+          },
+          "chart-name.alarm-name": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "integer",
```

Comment:
I think type here should be "number".

Reply:
Both number and integer are valid types but number is more general. If the `id` is an integer then the less general type is the better choice.

https://swagger.io/docs/specification/data-models/data-types/

Reply:
The `id` is formatted as %lu `(unsigned long)` taken from a structure that defines it as `uint32_t`

Reply:
I understood @amoss , thank you! I thought we needed to use `number` instead `integer`, because this is the most common type used in our documentation and I did not want to create doubts for the reader.